governance/polls/Ratification Poll for Adding Events Core Unit (EVENTS-001) - April 11, 2022.md

```diff
@@ -0,0 +1,83 @@
+
+**This is a binary vote.**
+- **You may vote for a single option.**
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+The community may vote in this poll to express support or opposition to Adding Events Core Unit (EVENTS-001) being accepted and implemented in the Maker Protocol.
+
+### MIP39c2-SP32: Adding events Core Unit (EVENTS-001)
+
+A brief summary of this proposal has been provided by the MIP Author and is shown below:
+
+*The Events Core Unit will handle planning, organization, and execution of MakerDAO branded events. Our vision is to use an exhaustive list of themes, visuals, set dressing, technology and branding to carry out events to the highest standards for the MakerDAO community.*
+
+Please review the links below to inform your position on this proposal before voting.
+* [Canonical Proposal Version](https://github.com/makerdao/mips/commit/1a75df31c022c3049a2f623ed3178a10457ff5fb)
+* [Latest Proposal Version](https://mips.makerdao.com/mips/details/MIP39c2SP32)
+* [Proposal Discussion Thread](https://forum.makerdao.com/t/mip39c2-sp32-adding-events-core-unit-events-001/13780)
+
+### MIP40c3-SP66: Events Core Unit DAI Budget (EVENTS-001)
+
+A brief summary of this proposal has been provided by the MIP Author and is shown below:
+
+*MIP40c3-SP66 adds the DAI budget for the Core Unit EVENTS-001: Events. This budget will allocate DAI to the CU. The one-off seed transfer will be 149,692 DAI with the monthly transfers being 62,371.50 (12 transfers total). The total CU budget amounts to 898,150 DAI annually.*
+
+Please review the links below to inform your position on this proposal before voting.
+* [Canonical Proposal Version](https://github.com/makerdao/mips/commit/f7588070945cab9ae27ead086e1126a4d343171a)
+* [Latest Proposal Version](https://mips.makerdao.com/mips/details/MIP40c3SP66)
+* [Proposal Discussion Thread](https://forum.makerdao.com/t/mip40c3-sp66-events-core-unit-dai-budget-events-001/13777)
+
+### MIP41c4-SP33: Events Core Unit Facilitator - Jon Perry (EVENTS-001)
+
+A brief summary of this proposal has been provided by the MIP Author and is shown below:
+
+*This sub proposal onboards Jon Perry as the Facilitator for EVENTS-001. As the facilitator, he will be responsible for ensuring the Events CU achieves its mandate of producing branded events for MakerDAO to the highest level of quality, transparency and professionalism.*
+
+Please review the links below to inform your position on this proposal before voting.
+* [Canonical Proposal Version](https://github.com/makerdao/mips/commit/bf522ee865929361524eddee332ef3a54d84bcb0)
+* [Latest Proposal Version](https://mips.makerdao.com/mips/details/MIP41c4SP33)
```

Comment:
Use this link: https://github.com/makerdao/mips/blob/bf522ee865929361524eddee332ef3a54d84bcb0/MIP41/MIP41c4-Subproposals/MIP41c4-SP33.md